models/QtiRunnerInitDataBuilder.php

```diff
@@ -303,4 +311,18 @@ private function getResponseCountsFromState(array $itemState): int
 
         return $responsesCount;
     }
+
+    /**
+     * @throws MissingOutcomeDeclarationMaxScore
+     */
+    private function getMaxScore($outcomes): ?float
+    {
+        foreach ($outcomes as $outcome) {
+            if ($outcome[self::OUTCOME_DECLARSTION_IDENTIFIER] === self::OUTCOME_DECLARATION_MAXSCORE) {
```

Comment:
is it possible `$outcome[self::OUTCOME_DECLARSTION_IDENTIFIER]` to not be set as well??
shouldn't we check it?

Reply:
https://github.com/oat-sa/extension-lti-test-review/pull/107/files#r929735835